### ISSUE TITLE
Installation with Contao 4.13 / Isotope 2.8 dev fails

### ISSUE THREAD

**Issue by 9teufel** [inspiredminds/contao-isotope-pdf-templates#3]
I'm getting this error while installing:

```
- Root composer.json requires inspiredminds/contao-isotope-pdf-templates ^2.1 -> satisfiable by inspiredminds/contao-isotope-pdf-templates[2.1.0, 2.1.1].
- contao/news-bundle[4.13.0, ..., 4.13.2] require symfony/config ^5.4 -> satisfiable by symfony/config[v5.4.0, v5.4.2, v5.4.3, v5.4.7].
- Conclusion: don't install symfony/config v5.4.7 (conflict analysis result)
- Root composer.json requires contao/news-bundle 4.13.* -> satisfiable by contao/news-bundle[4.13.0, 4.13.1, 4.13.2].
```

Thank you!

**Comment by southanx** [comment]
Same here with the new isotope 2.8...
Thank you for help!

**Comment by fritzmg** [comment]
Fixed in `2.1.2`.
